src/test/operators/sort_test.cpp

```diff
@@ -107,7 +114,9 @@ TEST_P(OperatorsSortTest, DoubleSortOfOneColumn) {
 TEST_P(OperatorsSortTest, DescendingSortOfOneColumn) {
   std::shared_ptr<Table> expected_result = load_table("resources/test_data/tbl/int_float_reverse.tbl", 2);
 
-  auto sort = std::make_shared<Sort>(_table_wrapper, ColumnID{0}, OrderByMode::Descending, 2u);
+  auto sort = std::make_shared<Sort>(
+      _table_wrapper, std::vector<SortColumnDefinition>{SortColumnDefinition{ColumnID{0}, OrderByMode::Descending}},
```

Comment:
I think it actually is necessary because of the `explicit`-ness of the `SortColumnDefinition` constructor.

Our way to this code was as follows:
- The previously [non-explicit constructor](https://github.com/bakoe/hyrise/blob/80232da1c62a902383ead90dd5678726f2241d52/src/lib/operators/sort.hpp#L21-L22) (and thus omitting of `std::vector<SortColumnDefinition>` in its usage, as proposed by you) [lead to an error in the Lint stage of the CI pipeline](https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-2005/4/pipeline#step-51-log-2) (`runtime/explicit `)
- Thus, [we made the `SortColumnDefinition` constructor `explicit`](https://github.com/hyrise/hyrise/commit/e220caf3e6fbc969559f991036e3a616c63a210f#diff-f0417e0e36c6f2046bc9907546366527L21-R21). This [lead to an error in the gccDebug stage of the CI pipeline](https://github.com/hyrise/hyrise/commit/e220caf3e6fbc969559f991036e3a616c63a210f#diff-f0417e0e36c6f2046bc9907546366527L21-R21) (`converting to 'opossum::SortColumnDefinition' from initializer list would use explicit constructor `)
- Thus, we made the usage of the `SortColumnDefinition` constructor explicit in 9937bc5750afb6a8d601465b7a9321b8ac8a5970.. 4b16c695a4bb701cadead86e35612d71ee28391f, leading to the existence of `std::vector<SortColumnDefinition>` here and in other places. 

Does this make sense?

Reply:
Hmpf. I would have thought that one `SortColumnDefinition` should be sufficient to satisfy the explicitness, but apparently not. Thanks for the explanation!